tests/test_package.py

```diff
@@ -43,6 +41,29 @@ def test_package_lookup(self, mock_requests, caplog, argument, monkeypatch):
         assert caplog.text
         assert argument[1] in caplog.text
 
+    @patch("tests.test_package.package.utils.install_packages")
+    @patch("tests.test_package.package.utils.parse_anaconda_packages")
+    def test_package_lookup_with_install_packages(
+        self, mock_package, mock_install, caplog
+    ):
+        mock_package = MagicMock(return_value=SplitRequirements([], []))
+        mock_install.return_value = (
+            True,
+            SplitRequirements(
+                [Requirement("snowflake-snowpark-python")],
+                [Requirement("some-other-package")],
+            ),
+        )
+
+        with caplog.at_level(logging.DEBUG, logger="snowcli.cli.snowpark.package"):
+            result = package.package_lookup("some-other-package", install_packages=True)
+
+        assert (
+            "The package some-other-package is supported, but does depend on the following Snowflake supported native "
+            'libraries you should include the following in your packages: [<Requirement: "snowflake-snowpark-python">]'
```

Comment:
> The package some-other-package is supported, but does depend on the following Snowflake supported native libraries you should include the following in your packages ...

`libraries you should` - It does look strange to me. It looks like `.` (dot) is missing after `libraries`. WDYT?

Reply:
Done